### PR TITLE
feat: fix new cow modal styles

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Fixes the new CoW Modal styles
Screenshot 2023-10-03 at 10 27 33

## Todo:
- CloseX (X close icon) click event to close doesn't work. Clicking outside the modal (backdrop area) **does** work.

### CODE REVIEW

**Comment by fairlighteth** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/cowswap-frontend/src/modules/trade/containers/TradeConfirmModal/index.tsx [hunk 1]
```diff
@@ -17,56 +17,65 @@ export interface TradeConfirmModalProps {
 }
 
 export function TradeConfirmModal(props: TradeConfirmModalProps) {
-  const { children } = props
+  const { children } = props;
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/modules/trade/containers/TradeConfirmModal/index.tsx:20]: Can you check your IDE/linter? It's adding `;` to this file for some reason

File: apps/cowswap-frontend/src/modules/trade/containers/TradeConfirmModal/index.tsx [hunk 2]
```diff
@@ -17,56 +17,65 @@ export interface TradeConfirmModalProps {
...
+      return <TransactionErrorContent message={error} onDismiss={onDismiss} />;
+    }
 
-        if (pendingTrade && permitSignatureState) {
-          // TODO: potentially replace TradeConfirmPendingContent completely with PermitModal
-          // We could use this not just for permit, but for any token, even already approved
-          const step = permitSignatureState === 'signed' ? 'submit' : 'approve'
+    if (pendingTrade && permitSignatureState) {
+      const step = permitSignatureState === 'signed' ? 'submit' : 'approve';
+      return (
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/modules/trade/containers/TradeConfirmModal/index.tsx:36]: Can you keep this TODO? It's a reference for me when I deal with this next

**Review by alfetopito — APPROVED** [review]
Looks great!!

**Pull request merged**